docs/en/running-node/open-node-tor-onion.md

```diff
@@ -29,23 +29,23 @@ in a secure and private way over Tor. Your node will be always ready w/o delays
 It also strenghtens Monero P2P network against DDoS attacks and network partitioning.
 
 **Open wallet inteface** (the "RPC") allows anyone to connect their wallets to Monero network through your node.
-This is useful for beginner users who don't run their own nodes yet.
+This is useful to users who don't run their own nodes.
 
-**Tor onion for wallet interface** is useful for wallet users connecting over Tor because it mitigates Tor exit nodes MiTM risks (which are very real). By connecting wallet to an onion service, no MiTM attack is realistic because within the Tor network connections are end-to-end TLS-ed.
+**Tor onion for wallet interface** is useful for wallet users connecting over Tor because it mitigates Tor exit nodes MiTM risks (which are very real). By connecting wallet to an onion service, no MiTM attack is realistic because within the Tor network connections are end-to-end encrypted.
 
 **Tor onion for P2P network** is useful for other full node users as it allows them to broadcast transactions over Tor (using `--tx-proxy` option).
 
 ## Assumptions
 
-You understand basic Linux administration. You seek Monero specific guidance.
+You understand basic Linux administration.
 
-You have root access to a Linux server with 2GB+ RAM and 120GB+ SSD (or 50GB+ for the pruned node version). This is current for Jan 2021.
+You have root access to a Linux server with 4GB+ RAM and available SSD storage of 512GB+ or 256GB+ (preferred) for the full or pruned node. As of 2024-09-01, the current blockchain size is ~220GB and ~85GB, respectively.
```

Comment:
food for thought: we could have a {{ macro }} for the current size on disk for a full / pruned node.. inside a 'note', e.g. 

Current blockchain sizes:
Fullnode size on disk: {{ lmdb_size_full }}
Pruned : {{ lmdb_size_pruned }}
Last updated : {{ lmdb_size_updated }}

Reply:
This is the top idea! Great!  :100: 

About the text: I  think there is a small mistake and 256+ and 512+(prefered) are in reverse order... or is preferred related to the pruned node?

Reply:
Related to the full and pruned 👍👍 


```
“Respectively” is an adverb meaning “in the order mentioned.” We use it when we want to emphasize the relationships between two pairs or items in two parallel lists (i.e., two lists that contain the same number of items). For example, we could use “respectively” as follows:

Paris and Berlin are the capital cities of France and Germany, respectively. 

```